src/net_processing.cpp

```diff
@@ -4412,9 +4462,10 @@ void PeerManagerImpl::MaybeSendAddr(CNode& node, Peer& peer, std::chrono::micros
     if (!peer.m_addr_relay_enabled) return;
 
     LOCK(peer.m_addr_send_times_mutex);
+    bool is_ibd = WITH_LOCK(::cs_main, return m_chainman.IsAnyChainInIBD());
```

Comment:
c4d90167f030547373715a44fc020d0da8461565
```suggestion
    const bool is_ibd{WITH_LOCK(::cs_main, return m_chainman.IsAnyChainInIBD())};
```